### PR TITLE
add necessary metadata for bruteforce

### PR DESCRIPTION
**Pull request by zgoldman-r7**
This PR adjusts the metadata for the ldap login scanner, adding defaults and adjusting the service and protocol values.

To test:

- spin up docker or windows instance with LDAP running (usable docker env details in `test/ldap`)
- use `ldap_login` module against target and ensure functionality still works

### CODE REVIEW

File: lib/metasploit/framework/login_scanner/ldap.rb [hunk 1]
```diff
@@ -11,19 +11,32 @@ class LDAP
         include Metasploit::Framework::LDAP::Client
         include Msf::Exploit::Remote::LDAP
 
+        # TODO: Verify if we need this additional metadata: https://github.com/rapid7/metasploit-framework/blob/master/lib/metasploit/framework/login_scanner/http.rb#L15-L23
```
Comment (adfoster-r7) [inline lib/metasploit/framework/login_scanner/ldap.rb:14]: We'll need to make sure that the ldaps functionality is implemented correctly here, similar to the above link

Comment (cgranleese-r7) [inline lib/metasploit/framework/login_scanner/ldap.rb:14]: I believe this can be removed.

```suggestion
```

**Comment by cgranleese-r7** [comment]
Tested and works as expected:
image

**Comment by cgranleese-r7** [comment]
## Release Notes
Adjusts the metadata for the ldap login scanner, adding defaults and adjusting the service and protocol values.

**Pull request merged**